MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -1570,30 +1430,25 @@ private void maybeShowPopup(MouseEvent e) {
                                 driverMenu.add(cbMenuItem);
                             }
                         }
-                        if (unit.canTakeMoreGunners()
-                                && person.canGun(unit.getEntity())) {
-                            cbMenuItem = new JCheckBoxMenuItem(
-                                    unit.getName());
-                            // TODO: check the box
+                        if (unit.canTakeMoreGunners() && person.canGun(unit.getEntity())) {
+                            cbMenuItem = new JCheckBoxMenuItem(unit.getName());
+                            cbMenuItem.setSelected(person.getUnitId() == unit.getId());
                             cbMenuItem.setActionCommand(makeCommand(CMD_ADD_GUNNER, unit.getId().toString()));
                             cbMenuItem.addActionListener(this);
                             gunnerMenu.add(cbMenuItem);
                         }
                         if (unit.canTakeMoreVesselCrew()
                                 && ((unit.getEntity().isAero() && person.hasSkill(SkillType.S_TECH_VESSEL))
                                     || ((unit.getEntity().isSupportVehicle() && person.hasSkill(SkillType.S_TECH_MECHANIC))))){
-                            cbMenuItem = new JCheckBoxMenuItem(
-                                    unit.getName());
-                            // TODO: check the box
+                            cbMenuItem = new JCheckBoxMenuItem(unit.getName());
+                            cbMenuItem.setSelected(person.getUnitId() == unit.getId());
```

Comment:
I'll stop asking about that reference equality, but there's more of these.

Reply:
Fourteen in total.